tests/pom.xml

```diff
@@ -231,6 +231,12 @@
       <groupId>org.eclipse.hono</groupId>
       <artifactId>hono-service-device-registry-base</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.jboss.slf4j</groupId>
+          <artifactId>slf4j-jboss-logmanager</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
FMPOV we should keep either this exclusion or the configuration of the failsafe plugin, but not both

Reply:
This must have come in via the extra "Exclude JBoss logging slf4j binding when running ITs" commit in the list. I didn't knowingly add it there. I'll rebase...

Reply:
Change is not in this PR any more now.